### ISSUE TITLE
Changing input and output language

### ISSUE THREAD

**Issue by tombarys** [LayBacc/roam-ai#2]
It would be great change the language in options. When using different language than english (Czech in my case) it kinda works, but behaves strange: it "understands" the prompt somehow but answer is in English and is not generated in a numbered list. Thanks!

**Comment by LayBacc** [comment]
agreed with the value of this. curious — does the [regular GPT-3 playground](https://beta.openai.com/playground) work well with Czech?

**Comment by tombarys** [comment]
Surprisingly it understands Czech perfectly but output is always in English.
